.changeset/serious-roses-stare.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-api-docs': minor
+---
+
+Link 'Register Existing API' button on API page to /catalog-import instead of /create .
```

Comment:
This isn't really the change from this packages point of view tbh. We've only renamed the `createComponent` external route to `registerComponent` (or `registerApi`). I think it could make sense to explain that this is done because it's now recommended to link it to the entity registration page rather than creation page.

Reply:
Ah, I see what you mean. Updated to be closer to your suggested changeset.